minio/Caddyfile

```diff
@@ -0,0 +1,8 @@
+your.public.com {
+    proxy / localhost:9000 {
+        proxy_header Host {host}
+        proxy_header X-Real-IP {remote}
+        proxy_header X-Forwarded-Proto {scheme}
```

Comment:
FYI, as of 0.9 beta these 3 lines can be replaced with one preset, "transparent".

```plain
proxy / localhost:9000 {
    transparent
}
```